cylc/rose/platform_utils.py

```diff
@@ -91,24 +93,36 @@ def get_platforms_from_task_jobs(
         Platform Dictionary.
     """
     _, _, flow_file = parse_id(flow, constraint='workflows', src=True)
```

Comment:
I think that this line is now surplus to requirements.

```suggestion
```

Reply:
Yes, looks like it

Reply:
Removed the assignments but kept the `parse_id` bit for now as a sanity check of the ID being passed in.